client/src/components/Button.tsx

```diff
@@ -15,18 +15,18 @@ const base = `
   before:block before:absolute before:top-0 before:left-0 before:bottom-0 before:right-0
   before:bg-transparent before:user-select-none before:-z-1 before:rounded-full
 
-  hover:before:bg-state-layers-light-primary/[.08]
-  focus:before:bg-state-layers-light-primary/[.12]
-  active:before:bg-state-layers-light-primary/[.12]
+  hover:before:bg-state-layers-primary/8
+  focus:before:bg-state-layers-primary/12
+  active:before:bg-state-layers-primary/12
 
-  disabled:bg-light-on-surface/[.12] disabled:text-light-on-surface/[.38] disabled:cursor-not-allowed
+  disabled:bg-sys-on-surface/12 disabled:text-sys-on-surface/[.38] disabled:cursor-not-allowed
```

Comment:
I am unsure on this but why do we need to include 'sys' here since the original name does not contain 'sys' in it?

Reply:
You're right, it should be `bg-state-layers-on-surface/12`